Sources/core/objects/Ammunationboxes.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Ammunationboxes.h"
+
+AmmunitionBox::AmmunitionBox(size_t id) : Item(id)
+{
+    SetSprite("icons/ammo.dmi");
+    SetState("357");
+
+    name = "Revolver Ammunition",
+	
+    bullets_ = 50;
+}
+
+bool AmmunitionBox::CheckBullets()
+{
+    if(bullets_>0)
+    {
+        return true;
+    }
+    if(bullets_<=0)
+    {
+        return false;
+    }
+}
```

Comment:
Second `if` is not needed. You can just write
```
if (bullets_ > 0)
{
    return true;
}
return false;
```
You should avoid unnecessary `if` and `else` where it is possible.